docs/js/features/odata/generate-client.md

```diff
@@ -0,0 +1,89 @@
+---
+id: generator-js-sdk
+title: Generate an OData client for JavaScript
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: Generate an OData client for JavaScript
+keywords:
+- sap
+- cloud
+- sdk
+- odata
+- JavaScript 
+- TypeScript
+- generate
+author: Charles Dubois
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Generate a typed OData client with the SAP Cloud SDK generator
+
+The generator allows you to generate custom OData clients libraries for OData services. You can then easily access these services via the client libraries.
+
+The SAP Cloud SDK generator can both be used as a command line interface (CLI) and programmatically.
+
+All you need to use it is a service metadata specification in the `EDMX` format (file ending can be `.edmx` or `.xml`).
+
+## Installation
+
+Run this command in your project's terminal to install the generator globally.
+```sh
+npm install -g @sap-cloud-sdk/generator
+```
+
+:::note
+The generator is included in the SAP Cloud SDK CLI, if you have it installed you don't need to install the generator separately.
```

Comment:
Sorry this sentence still sounds weird. Maybe split it into two.
```suggestion
The generator is included in the SAP Cloud SDK CLI. If you installed the CLI, you don't need to install the generator separately.
```

Reply:
BTW, I never tried the generator from the CLI myself, but maybe you can try it out and in case there is a difference in how this is invoked add it to the usage section below. ( I could imagine that it is something like `sap-cloud-sdk generate-odata-client ...`).